resources/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM ubuntu:18.04 as dev
 
```

Comment:
For b40aea07e99ef45a794df89f96c717f3df846a35 please prefix first line of commit message with "build:" and drop trailing full-stop.

Please remove the Gerrit Change-Id.

Reply:
Done for all commits.